app/helpers/application_helper/button/basic.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationHelper::Button::Basic < Hash
   include ActionView::Helpers::TextHelper
 
-  delegate :role_allows?, :to => :@view_context
+  delegate :role_allows?, :rbac_common_feature_for_buttons, :to => :@view_context
```

Comment:
@bmclaughlin i think we should keep this file the way it is and not have it call rbac_common_features_for_buttons for all toolbar buttons. let's add a new button file that is specific to tenant buttons, see the attached patch with suggested changes.
[restore_project_add_button_.patch.zip](https://github.com/ManageIQ/manageiq/files/583788/restore_project_add_button_.patch.zip)